tests/unit/cbprintf/main.c

```diff
@@ -584,7 +584,7 @@ static void test_d_length(void)
 	reset_out();
 	rc = rawprf("/%Ld/", max);
 	zassert_equal(rc, 5, "len %d", rc);
-	zassert_equal(strncmp("/%Ld/", buf, rc), 0, NULL);
+	zassert_equal(strncmp("/%lld/", buf, rc), 0);
```

Comment:
Please leave `%Ld` here as is. From what I can see, this is intentionally testing an invalid format specifier.

Reply:
Yes, you're right. I didn't revert this patch yet, because I am looking for a help with suppressing the checkpatch warning about this format.